e2e/nomostest/testing/wrapper.go

```diff
@@ -16,22 +16,15 @@ package testing
 
 import (
 	"strings"
-	"time"
+	"testing"
 
 	"kpt.dev/configsync/e2e"
 )
 
-const errorPrefix = "ERROR:"
```

Comment:
Do we still need the `ERROR:` prefix? It was added to make test failures more obvious. It was broken when timestamp was added. We could make it work again.

Reply:
I don't mind having the error prefix, but where do you propose injecting it?

Reply:
Updated this to keep the wrapper abstraction with a slightly different implementation so that we can keep the injected prefixes

Reply:
I use "ERROR: "a lot to search for failures in the log.

Reply:
nt.Must -> wrapper -> testing.T. Isn't it also a two tiers of Helper function?

Reply:
With embedding, a call to `wrapper.Helper` now invokes the real `Helper` method - so when it inspects the stack trace it adds the correct function name